crates/biome_cli/src/diagnostics.rs

```diff
@@ -229,11 +230,18 @@ pub struct IncompatibleEndConfiguration {
 
 #[derive(Debug, Diagnostic)]
 #[diagnostic(
-    category = "internalError/io",
     severity = Error,
-    message = "No files were processed in the specified paths."
+    message = "No files were processed in the specified paths.",
+    advice = "Check your biome.json to ensure the paths are not ignored by the configuration.",
```

Comment:
Is it difficult in this context to find out if the file is called `biome.json` or `biome.jsonc`? Otherwise we can also say `biome.json/biome.jsonc`

Reply:
Chose to say biome.json or biome.jsonc for now. I couldn't see an easy way to find the loaded configuration path. I could use `load_configuration`, but it's a little heavy operation for just emitting one diagnostic. Plus, the `#[diagnostic]` proc macro don't supported markup in advices.